diff -ru '--exclude=.git' pristine/src/gestures.h working/src/gestures.h
--- pristine/src/gestures.h	2026-09-10 22:05:09.962685431 +0400
+++ working/src/gestures.h	2026-09-10 22:05:09.985686021 +0400
@@ -51,7 +51,7 @@
     Q_OBJECT
 
 public:
-    static constexpr double s_minimumDelta = 200;
+    static constexpr double s_minimumDelta = 20;
 
     explicit SwipeGesture(uint32_t fingerCount);
     ~SwipeGesture() override;
