diff -ru '--exclude=.git' pristine/applets/systemtray/qml/main.qml working/applets/systemtray/qml/main.qml
--- pristine/applets/systemtray/qml/main.qml	2026-09-08 04:56:11.391758756 +0400
+++ working/applets/systemtray/qml/main.qml	2026-09-08 04:56:11.394758797 +0400
@@ -192,14 +192,14 @@
                 // The icon size to display when not using the auto-scaling setting
                 readonly property int smallIconSize: Kirigami.Units.iconSizes.smallMedium
 
-                readonly property bool autoSize: Plasmoid.configuration.scaleIconsToFit
+                readonly property bool autoSize: Kirigami.Settings.tabletMode || Plasmoid.configuration.scaleIconsToFit
 
                 readonly property int gridThickness: root.vertical ? root.width : root.height
                 // Should change to 2 rows/columns on a 56px panel (in standard DPI)
                 readonly property int rowsOrColumns: autoSize ? 1 : Math.max(1, Math.min(count, Math.floor(gridThickness / (smallIconSize + Kirigami.Units.smallSpacing))))
 
                 // Add margins only if the panel is larger than a small icon (to avoid large gaps between tiny icons)
-                readonly property int cellSpacing: Kirigami.Units.smallSpacing * Plasmoid.configuration.iconSpacing
+                readonly property int cellSpacing: Kirigami.Units.smallSpacing * (Kirigami.Settings.tabletMode ? 4 : Plasmoid.configuration.iconSpacing)
                 readonly property int smallSizeCellLength: gridThickness < smallIconSize ? smallIconSize : smallIconSize + cellSpacing
 
                 cellHeight: {
